### DISCUSSION TITLE
Last update wallpapers does not render

### DISCUSSION THREAD

**Discussion by crglipi** [mylinuxforwork/dotfiles#542] [Q&A]
After the last update the wallpaper just turned black and the menu options to change wallpaper also dosen´t open or work- The terminal changes the colour and when I manual try hyprpaper from the terminal it renders the wallpaper but only when the terminal it is open (even if I type: "hyprpaper &"). Please help I love your work ml4w and your skills are amazing. Thanks.

**Comment by LBTRS1911** [comment]
Having the same exact issue. Updated EndeavourOS and now get just a black screen with cursor after log in sddm. I can open a terminal but have no wallpaper or bar, just a black screen. Poked around and don't even know where to start.

Opened the terminal and ran the ml4w-hyprland-setup script thinking I needed to update to the new version but that didn't help.

**Reply by SSYH** [reply]
same

**Comment by AlexsDarn** [comment]
Hi, with the new update it was fixed
